### ISSUE TITLE
Config Options for Patient Photo

### ISSUE THREAD

**Issue by robertdown** [openemr/openemr#4840]
### Describe the problem

Patient Photos that are uploaded do not appear to undergo any processing. A few config options could help this along. Generally, you'd want to size down anything over a certain size and possible drop down the quality of the image. This would improve load times and reduce file size on the file system.

**Comment by robertdown** [comment]
bump

**Comment by ajmeese7** [comment]
I agree with the OP, this issue should not be closed as stale as it is still relevant.
